src/OpenTelemetry.AutoInstrumentation/Logging/CustomLogger.cs

```diff
@@ -156,34 +156,34 @@ public void Error<T>(Exception exception, string messageTemplate, T property, [C
     public void Error(Exception exception, string messageTemplate, object[] args, [CallerLineNumber] int sourceLine = 0, [CallerFilePath] string sourceFile = "")
         => Write(LogLevel.Error, exception, messageTemplate, args, sourceLine, sourceFile);
 
-    private void Write<T>(LogLevel level, Exception exception, string messageTemplate, T property, int sourceLine, string sourceFile)
+    private void Write<T>(LogLevel level, Exception? exception, string messageTemplate, T property, int sourceLine, string sourceFile)
```

Comment:
Should `Exception` be nullable here? there is already an overload without exception. I would force it not to be `null` here.

Reply:
It is private method, needed only due to some optimization ( < 3 additional properties).
All of them eventually leads to WriteImpl.

All public methods have Exception non-nullable.